node-lib/src/config_files/p2p.rs

```diff
@@ -69,6 +69,9 @@ pub struct P2pConfigFile {
     pub ban_threshold: Option<u32>,
     /// Duration of bans in seconds.
     pub ban_duration: Option<u64>,
+    /// Maximum acceptable time difference between this node and the remote peer.
+    /// If a large difference is detected, the peer will be disconnected.
+    pub max_clock_diff: Option<u64>,
```

Comment:
Since we are not using a `Duration` here, I would specify in the doc comment that this value is in seconds (similar to how it's documented for `ban_duration`).